examples/morpheusvm/cmd/morpheus-cli/cmd/resolutions.go

```diff
@@ -52,8 +52,9 @@ func handleTx(tx *chain.Transaction, result *chain.Result) {
 	status := "⚠️"
 	if result.Success {
 		status = "✅"
-		switch action := tx.Action.(type) { //nolint:gocritic
-		case *actions.Transfer:
+		switch tx.Action.GetTypeID() { //nolint:gocritic
+		case (&actions.Transfer{}).GetTypeID():
```

Comment:
Ah, I was thinking you just would do:
```
case TransferTypeID:
```

Reply:
I thought about doing it but it makes potentially use a declared ID not used by any Type.
The both were possible.